### PR TITLE
Fix  error loading namespace i18n

### PR DESCRIPTION
**Pull request by Nancy-Badran**
**Description of your changes:**

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: backend/src/apiserver/resource/resource_manager_util_test.go [hunk 1]
```diff
@@ -188,7 +188,7 @@ spec:
       command:
       - python
       - -c
-      image: python:alpine3.6
+      image: python:alpine
```
Comment (Jose-Matsuda) [inline backend/src/apiserver/resource/resource_manager_util_test.go:191]: is there a reason for this image not being pinned to a version? same as on line 288

File: backend/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 # Use golang:1.13.1-stretch to keep GLIBC at 2.24 https://github.com/gotify/server/issues/225
 FROM golang:1.13.1-stretch as builder
-RUN apt-get update && apt-get install -y cmake clang musl-dev openssl
+RUN apt-get update && apt-get install -y cmake clang musl-dev 
+
+
+RUN apt-get update
+RUN apt-get install -y build-essential openssl zlib1g-dev libcppunit-dev git subversion wget && rm -rf /var/lib/apt/lists/*
+
+RUN var = openssl ARGO_VERSIONRUN echo ${var}
```
[the remark this answers is not in the document]
Comment (Nancy-Badran) [reply backend/Dockerfile:10]: OOPs, It was testing, forget to return to original

File: frontend/src/i18n.ts [hunk 3]
```diff
@@ -1,26 +1,60 @@
...
 i18n
-  .use(XHR)
-  .use(Backend)
   .use(LanguageDetector)
   .use(initReactI18next)
   .init({
-    ns: ['common', 'home', 'artifacts', 'executions', 'experiments', 'pipelines'],
-    defaultNS: 'common',
-    fallbackLng: ['en', 'fr'],
+    detection: options,
```
[the remark this answers is not in the document]
Comment (Nancy-Badran) [reply frontend/src/i18n.ts:49]: As you see,  we passed the LanguageDetector object to use function of i18next. But this is not enough, we have to define another object to detect the mechanisme. https://codetain.com/blog/how-to-translate-react-application-with-react-i18next-part-2

File: backend/src/apiserver/resource/resource_manager_util_test.go (hunk 1 above)
Comment (Nancy-Badran) [reply backend/src/apiserver/resource/resource_manager_util_test.go]: fixed

**Review by Jose-Matsuda — APPROVED** [review]
LGTM

**Pull request merged**